Formula/openimageio.rb

```diff
@@ -47,7 +47,7 @@ def install
       -DCCACHE_FOUND=
       -DEMBEDPLUGINS=ON
       -DUSE_FIELD3D=OFF
```

Comment:
Unrelated to current failure, but we may want to build with Homebrew `pugixml` instead:
```suggestion
      -DUSE_EXTERNAL_PUGIXML=ON
      -DUSE_FIELD3D=OFF
```

Ref: https://github.com/OpenImageIO/oiio/blob/v2.3.18.0/INSTALL.md#optional-dependencies----features-may-be-disabled-if-not-found
> - We use PugiXML for XML parsing. There is a version embedded in the OIIO tree, but if you want to use an external, system-installed version (as may be required by some software distributions with policies against embedding other projects), then just build with `-DUSE_EXTERNAL_PUGIXML=1`. Any PugiXML >= 1.8 should be fine (we have tested through 1.11).